tools/releases/docker.sh

```diff
@@ -32,9 +35,31 @@ function push() {
   docker_login
 
   for component in ${KUMA_COMPONENTS}; do
-    msg "Pushing $component:$KUMA_VERSION ..."
-    docker push "${KUMA_DOCKER_REPO_ORG}/${component}:${KUMA_VERSION}"
-    msg_green "... done!"
+    for arch in ${BUILD_ARCH}; do
+      msg "Pushing $component:$KUMA_VERSION-$arch ..."
+      docker push "${KUMA_DOCKER_REPO_ORG}/${component}:${KUMA_VERSION}-${arch}"
+      msg_green "... done!"
+    done
+  done
+
+  docker_logout
+}
+
+# allows to push many arch types as one tag
+function manifest() {
```

Comment:
Could this just all be in `push`?

Reply:
I wanted to split it because it is a different operation, but if that makes more sense I can change it